build.sbt

```diff
@@ -1558,7 +1558,7 @@ lazy val security = (project in file("security"))
       )
     )
   )
-  .dependsOn(utilsInternal, httpUtils, testUtils % "it,test")
+  .dependsOn(utilsInternal, httpUtils, testUtils % "it,test", restmodel)
```

Comment:
@arkadius we have added `restmodel` dependency to the `security` module (because we moved some security-related model classes to `restmodel`). If you see any downsides, please let us know. IMO this should be not a problem.

Reply:
Now it's good